bitrise.yml

```diff
@@ -4,7 +4,7 @@ default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 app:
   envs:
     - SAMPLE_APP_URL: https://github.com/bitrise-samples/sample-apps-flutter-ios-android.git
-    - SAMPLE_APP_BRANCH: "manual"
+    - SAMPLE_APP_BRANCH: master
```

Comment:
It was failing for me locally for some reason. `manual` is an old branch, while `master` has been updated a few times.

Reply:
wasn't there anything special about that branch?

Reply:
It has some iOS code signing changes, so I guess that branch is used in other Flutter steps, but that's irrelevant here